package.json

```
@@ -47,19 +51,25 @@
   "devDependencies": {
     "ajv": "^8.17.1",
     "ajv-cli": "^5.0.0",
+    "bio-dts": "^0.12.0",
     "chai": "^4.5.0",
+    "del-cli": "^6.0.0",
     "eslint": "^9.17.0",
     "eslint-plugin-bpmn-io": "^2.0.2",
     "fast-glob": "^3.3.3",
     "mocha": "^10.8.2",
     "npm-run-all": "^4.1.2",
-    "rollup": "^4.30.1"
+    "rollup": "^4.30.1",
+    "ts-expect": "^1.3.0",
+    "typescript": "^5.7.2"
   },
   "dependencies": {
     "min-dash": "^4.2.1"
   },
+  "types": "lib",
```

Typically indicates the main typing file, hence should be:

```suggestion
  "types": "lib/index.d.ts",
```

---

agree😀